server/lib/realtime/subscribers_notification.ex

```diff
@@ -95,6 +95,9 @@ defmodule Realtime.SubscribersNotification do
               end
             end)
           end
+
+        "TRUNCATE" ->
+          nil
```

Comment:
Deliberately empty - `TRUNCATE` notification is done in "Shout to specific table" above.